PlayDate/Views/Cells/MatchTableViewCell.swift

```diff
@@ -8,16 +8,89 @@
 import UIKit
 
 class MatchTableViewCell: UITableViewCell {
+    
+    static let identifier = "MatchTableViewCell"
+    
+//    static let appUser: User = {
+//        let user = User(userId: <#T##Int#>, name: <#T##String#>, bio: <#T##String#>, birthday: <#T##String#>, age: <#T##Int#>, pets: <#T##[Pet]#>)
+//        return user
+//    }()
+    
+    private let profileImageView: UIImageView = {
+        let imageView = UIImageView()
+        imageView.image = UIImage(named: "AppIcon")
+        imageView.contentMode = .scaleAspectFill
+        imageView.backgroundColor = .systemTeal
+        imageView.layer.cornerRadius = 50
+        imageView.layer.masksToBounds = true
+        
+        return imageView
+    }()
+    
+    private let petNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 21, weight: .semibold)
+        //label.text = appUser.pets
+        return label
+    }()
+    
+    private let ownerNameLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 19, weight: .regular)
+        //label.text = appUser.name
+        return label
+    }()
+    
+    private let descriptionLabel: UILabel = {
+        let label = UILabel()
+        label.font = .systemFont(ofSize: 15, weight: .regular)
+        label.textColor = .lightGray
+        label.numberOfLines = 0
+        //label.text = appUser.bio
+        return label
+    }()
+    
+//    private let noMatchesLabel: UILabel = {
+//        let label = UILabel()
+//        label.text = "No Matches"
+//        label.textAlignment = .center
+//        label.textColor = .gray
+//        label.font = .systemFont(ofSize: 21, weight: .medium)
+//
+//        return label
+//    }()
```

Comment:
Same thing here?

Reply:
This was something I added early on while I was following along with one of the iOSAcademy videos. The idea was to have something there displayed when a user had no matches available, but I never actually got around to implementing it. I wasn't sure if we would need it in the future so I commented it out just in case. I can definitely take it out to clean up the code.